src/base.cpp

```diff
@@ -623,7 +623,7 @@ Matrix::identityError(void)
 void
 correctPathCase(char *filename)
 {
-#ifndef _WIN32
+#if !defined(_WIN32) && !defined(RW_PS2)
```

Comment:
What's the motivation behind this change?

(I'm not sure if PS2 is case sensitive)

---

Also note that this is still called below (also conditional, which looks like bad design in the original code)

Reply:
How does the original premake deal with this then?

Reply:
premake does not deal with it. No exceptions. https://github.com/aap/librw/blob/b1c3c1dca8c5c2244ba7ecfacd50310f27a31b8e/premake5.lua#L64-L69

Reply:
https://github.com/ps2dev/ps2sdk/issues/76

Reply:
I don't know what changed in my development environment (apart from disabling the optimizations for PS2),
but this error has disappeared..